src/JWTGuard.php

```diff
@@ -501,6 +533,33 @@ protected function fireFailedEvent($user, array $credentials)
         ));
     }
 
+    /**
+     * Fire the authenticated event if the dispatcher is set.
```

Comment:
> if the dispatcher is set.

This isn't correct: the dispatcher is always present

Reply:
done